Fix missing import:

    ../pam/pam_fprintd.c: In function 'now':
    ../pam/pam_fprintd.c:76:3: error: implicit declaration of function 'clock_gettime' [-Werror=implicit-function-declaration]
       76 |   clock_gettime (CLOCK_MONOTONIC, &ts);

--- a/pam/pam_fprintd.c
+++ b/pam/pam_fprintd.c
@@ -36,6 +36,7 @@
 #include <signal.h>
 #include <sys/signalfd.h>
 #include <poll.h>
+#include <time.h>
 
 #define PAM_SM_AUTH
 #include <security/pam_modules.h>
